### PR TITLE
chore: reprovision

### PR DESCRIPTION
**Pull request by fabiosantoscode**
Get this built on travis!

Paused component-google-search on gocd

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -113,49 +113,53 @@
...
-    "phantomjs-prebuilt": "^2.1.7",
+    "npm-run-all": "^1.5.3",
+    "phantomjs-prebuilt": "^2.1.5",
     "postcss-cli": "^2.5.1",
     "postcss-cssnext": "^2.5.0",
     "postcss-import": "^8.0.2",
     "postcss-reporter": "^1.3.3",
     "postcss-url": "^5.1.1",
-    "react-addons-test-utils": "^0.14.8",
+    "react-addons-test-utils": "^0.14.7",
```
Comment (gcedo) [inline package.json]: regression

File: package.json [hunk 2]
```diff
@@ -113,49 +113,53 @@
   },
   "devDependencies": {
     "@economist/doc-pack": "^1.0.6",
-    "@economist/provision-react-component": "1.7.3",
+    "@economist/provision-react-component": "2.1.0",
     "babel": "^5.8.34",
-    "babel-polyfill": "^6.7.4",
+    "babel-polyfill": "^6.6.1",
```
Comment (gcedo) [inline package.json]: regression

File: package.json [hunk 3]
```diff
@@ -113,49 +113,53 @@
...
     "chai-spies": "^0.7.1",
+    "coveralls": "^2.11.9",
     "enzyme": "^2.1.0",
-    "eslint": "^2.7.0",
+    "eslint": "^2.8.0",
     "eslint-config-strict": "^8.5.1",
     "eslint-config-strict-react": "^8.0.1",
     "eslint-plugin-filenames": "^0.2.0",
-    "eslint-plugin-react": "^4.3.0",
+    "eslint-plugin-react": "^4.2.1",
```
Comment (gcedo) [inline package.json]: regression

File: package.json [hunk 4]
```diff
@@ -113,49 +113,53 @@
...
     "karma-coverage": "^0.5.5",
     "karma-mocha": "^0.2.2",
     "karma-mocha-reporter": "^2.0.0",
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-sauce-launcher": "^0.3.1",
+    "lcov-result-merger": "^1.1.0",
     "live-server": "^0.9.2",
     "mocha": "^2.4.5",
     "npm-assets": "^0.1.1",
-    "npm-run-all": "^1.7.0",
```
Comment (gcedo) [inline package.json]: regression

File: package.json [hunk 5]
```diff
@@ -132,18 +133,20 @@
...
     "karma-mocha-reporter": "^2.0.0",
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-sauce-launcher": "^0.3.1",
+    "lcov-result-merger": "^1.1.0",
     "live-server": "^0.9.2",
     "mocha": "^2.4.5",
     "npm-assets": "^0.1.1",
     "npm-run-all": "^1.7.0",
-    "phantomjs-prebuilt": "^2.1.7",
+    "phantomjs-prebuilt": "^2.1.5",
```
Comment (gcedo) [inline package.json]: regression

File: package.json [hunk 6]
```diff
@@ -152,10 +155,11 @@
     "react-addons-test-utils": "^0.14.8",
     "react-dom": "^0.14.7",
     "semantic-release": "^4.3.5",
-    "stylelint": "^5.3.0",
+    "stylelint": "^5.0.1",
```
Comment (gcedo) [inline package.json]: regression

File: package.json (hunk 5 above)
Comment (keithamus) [inline package.json]: @fabiosantoscode are you using PRC@3? Try using PRC@3 😄

Comment (fabiosantoscode) [inline package.json]: Hmm. I always install latest PRC. This time it didn't work, weird.

Comment (fabiosantoscode) [inline package.json]: Ah, it was just released :p

Comment (gcedo) [inline package.json]: PRC 3 uses React 15...

**Comment by gcedo** [comment]
@fabiosantoscode, can you rebase all those fix into `chore: provision`?

**Comment by fabiosantoscode** [comment]
@gcedo Easier to do a squashing merge?

**Comment by fabiosantoscode** [comment]
Just squashed it, updated PRC anyway. Hail React 15!

**Comment by fabiosantoscode** [comment]
Nope, PRC@3 doesn't work just yet. I do NOT miss EPEERINVALID.

**Comment by gcedo** [comment]
LGTM 👍

**Pull request merged**